### ISSUE TITLE
[BUG] Box-shadow gets cut off

### ISSUE THREAD

**Issue by hjorte-bd** [kirbydesign/designsystem#3171]
## Describe the bug
Box-shadow on gets cuts off by kirby-section-header.
 
## Describe how to reproduce the bug
1. Use kirby-button inside kirby-section-header
2. Shadow will be cut on top, on the left and bottom like in the first screenshot

## Which Kirby version was used?
8.8.0

## What was the expected behavior?
Box-shadow should extend outside shadow-dom element, like in the second screenshot

## Add any screenshots
![Screenshot 2023-07-19 at 14 49 13](https://github.com/kirbydesign/designsystem/assets/110398431/08c24643-e312-4e11-8e4b-791c29aa18b2)
![Screenshot 2023-07-19 at 15 03 18](https://github.com/kirbydesign/designsystem/assets/110398431/d5ad0bf2-b318-4a2b-80b5-ee90c092bcec)

## Please complete the following information:
- OS: MacOS
- Browser: Edge
- Browser version: 114.0.1823.67

## Are there any additional context?
It cannot be overriden from outside the shadow dom. Perhaps a variable thats allows overflow to be set to "visible" could be a solution

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
